ui/src/containers/BasicQueryBuilder/ActorDiscrete/Criteria.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Thank you for taking the time to make these additional components and not hacking the existing ones 👍